addons/flannel/template/base/install.sh

```diff
@@ -168,6 +168,6 @@ function flannel_kubeadm() {
         sed -i "/serviceSubnet/ s/.*/${pod_cidr_range_line}\n&/" /opt/replicated/kubeadm.conf
     fi
 
-    sudo kubeadm init phase upload-config kubeadm --config=/opt/replicated/kubeadm.conf
-    sudo kubeadm init phase control-plane controller-manager --config=/opt/replicated/kubeadm.conf
+    kubeadm init phase upload-config kubeadm --config=/opt/replicated/kubeadm.conf
+    kubeadm init phase control-plane controller-manager --config=/opt/replicated/kubeadm.conf
 }
```

Comment:
Should we not also add an testgrid for this one?

Reply:
There is a new on-pr testgrid for this